src/Collection/Collection.service.spec.ts

```diff
@@ -47,4 +54,106 @@ describe('Collection service', () => {
       })
     })
   })
+
+  describe('when checking if an address is a TPW manager', () => {
+    const service = new CollectionService()
+
+    afterAll(() => {
+      jest.restoreAllMocks()
+    })
+
+    describe('when an address belongs to the TPW_MANAGER_ADDRESSES env var', () => {
+      let manager: string
+      beforeEach(() => {
+        manager = '0x123123'
+        jest
+          .spyOn(env, 'get')
+          .mockReturnValueOnce(`0x555,${manager},0x444,0x333`)
+      })
+
+      it('should return true', async () => {
+        expect(await service.isTPWManager('', manager)).toBe(true)
+        expect(env.get).toHaveBeenCalledWith('TPW_MANAGER_ADDRESSES', '')
+      })
+    })
+
+    describe('when an address does not belong to the TPW_MANAGER_ADDRESSES env var', () => {
+      let urn: string
+      let manager: string
+
+      beforeEach(() => {
+        urn = 'some:valid:urn'
+        manager = '0x123123'
+        jest.spyOn(env, 'get').mockReturnValueOnce(`0x555,0x444,0x333`)
+      })
+
+      describe('when thegraph has a urn with the address as manager', () => {
+        beforeEach(() => {
+          ;(thirdPartyAPI.isManager as jest.Mock).mockReturnValueOnce(true)
+        })
+
+        it('should return true', async () => {
+          expect(await service.isTPWManager(urn, manager)).toBe(true)
+          expect(thirdPartyAPI.isManager).toHaveBeenCalledWith(urn, manager)
+          expect(env.get).toHaveBeenCalledWith('TPW_MANAGER_ADDRESSES', '')
+        })
+      })
+
+      describe('when thegraph does not has a urn with the address as manager', () => {
+        beforeEach(() => {
+          ;(thirdPartyAPI.isManager as jest.Mock).mockReturnValueOnce(false)
+        })
+
+        it('should return true', async () => {
+          expect(await service.isTPWManager(urn, manager)).toBe(false)
+          expect(thirdPartyAPI.isManager).toHaveBeenCalledWith(urn, manager)
+          expect(env.get).toHaveBeenCalledWith('TPW_MANAGER_ADDRESSES', '')
+        })
+      })
+    })
+  })
+
+  describe('when getting the database TPW collections', () => {
+    const service = new CollectionService()
```

Comment:
It might be better to have this initialized in a before each in case we decide to make it stateful.